core/main/src/broker/endpoint_broker.rs

```diff
@@ -502,7 +540,7 @@ pub trait EndpointBroker {
     }
 
     /// Generic method which takes the given parameters from RPC request and adds rules using rule engine
-    fn apply_request_rule(rpc_request: &BrokerRequest) -> Result<Value, RippleError> {
+    fn apply_request_rule(rpc_request: &BrokerRequest) -> Result<Value, JqError> {
```

Comment:
Can we keep them as Ripple Error instead of JqError

Ripple error is already an enum which does the same for Deny Reason

Reply:
RippleError is overloaded imho, which is why i leaned into this more rust idiomatic apporach.